android-offline-maps/app/src/main/java/com/example/mapbox_offline/TileRegionDownloadView.kt

```diff
@@ -0,0 +1,151 @@
+package com.example.mapbox_offline
+
+import android.util.Log
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material.icons.filled.CheckCircle
+import androidx.compose.material3.Button
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Divider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.mapbox.common.MapboxOptions
+import com.mapbox.common.TileStore
+import com.mapbox.common.TileRegion
+import com.mapbox.maps.mapsOptions
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun TileRegionDownloadScreen(
+    viewModel: TileRegionDownloadViewModel = TileRegionDownloadViewModel(),
```

Comment:
We could access TileRegionDownloadViewModel using [viewModel](https://developer.android.com/reference/kotlin/androidx/lifecycle/viewmodel/compose/package-summary#viewModel(androidx.lifecycle.ViewModelStoreOwner,kotlin.String,kotlin.Function1)) or [by viewModels()](https://developer.android.com/reference/kotlin/androidx/activity/package-summary#(androidx.activity.ComponentActivity).viewModels(kotlin.Function0,kotlin.Function0)) Kotlin property delegate.